core/jobs/batch_jobs/story_description_validation_jobs.py

```diff
@@ -0,0 +1,91 @@
+# coding: utf-8
```

Comment:
Lets name this file _story_validation_jobs.py_

Reply:
done